src/System.Text.JsonLab/System/Text/Json/JsonReaderException.cs

```diff
@@ -7,15 +7,15 @@ namespace System.Text.JsonLab
     public class JsonReaderException : Exception
     {
         //TODO: Consider adding line number and position to the message itself
-        public JsonReaderException(string message, long lineNumber, long bytePosition) : base(message)
+        public JsonReaderException(string message, long lineNumber, long lineBytePosition) : base(message)
```

Comment:
Is there a code sample proving that we can write helpers on top of the position APIs that would allow higher level code to compute the information that the callers actually want: the text around the place where the error occurred.

Reply:
Not yet, I will write something up. In any case, it requires the caller to have access to all the data so they can seek to the appropriate location and encode the byte position to character position.